lib/holiday-stops/src/test/scala/com/gu/holiday_stops/CreditCalculatorSpec.scala

```
@@ -0,0 +1,45 @@
+package com.gu.holiday_stops
+
+import java.time.LocalDate
+
+import io.circe.generic.auto._
+import io.circe.parser.decode
+import org.scalatest._
+
+import scala.io.Source
+
+class CreditCalculatorSpec extends FlatSpec with Matchers with EitherValues {
```

This is a kind of unit test I would prefer over protocol wiring tests if we had to make a choice between the two, because here we are testing key piece of business logic which would be silently failing as logic error as opposed to wiring testing which would very likely surface on the first UAT if not even at compile time.